gallery/README.md

```diff
@@ -72,3 +72,30 @@ more details):
 ```
 flutter pub run grinder update-code-segments
 ```
+
+## Creating a new release (for Flutter org members)
+
+1. Bump the version number up in the `pubspec.yaml`. Use semantic versioning to determine 
+   which number to increment. For example `2.2.0+020200` should become `2.3.0+020300`.
+   
+2. Create a tag on the `master` branch of this repo in the form of `gallery-v2.3`.
+	* `git tag gallery-v2.3`
+	* `git push --tags`
+
+3. Publish the web release (using the peanut package).
```

Comment:
Consider linkifying the name "peanut" to point to https://pub.dev/packages/peanut.